.github/workflows/integration-test.yaml

```diff
@@ -69,6 +69,42 @@ jobs:
     with:
       group_dir: ${{ matrix.group }}
     secrets: inherit
+  merge-logs:
+    # download all artifacts across containers
+    needs: [pipeline-ci]
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+    - uses: actions/download-artifact@v4
```

Comment:
Can you pin this version with the latest tag SHA like the others?

Reply:
Have added latest SHA tag!